lib/src/command/outdated.dart

```diff
@@ -66,10 +73,16 @@ class OutdatedCommand extends PubCommand {
     argParser.addFlag('pre-releases',
         defaultsTo: false, help: 'Alias of prereleases.', hide: true);
 
+    argParser.addFlag('show-all',
+        defaultsTo: false,
+        help: 'Include dependencies that are already fullfilling --mode.');
```

Comment:
`negatable: false`

There is no reason to have a `--no-show-all` flag, it's the default. Does it really make sense to pass this flag explicitly?

Reply:
If you want to ensure the default doesn't change under you.

You could say this for several of our boolean flags. I think it is nice to be able to be explicit.